run-development.sh

```diff
@@ -64,7 +64,16 @@ fi
 # Start server
 ##############
 
-cd /usr/src/build/
-/usr/src/app/node_modules/.bin/babel-node \
-    --inspect="0.0.0.0:9229" \
-    ./app.js
+cd /usr/src/dist/
+if [ "$NO_BABEL_NODE" == "true" ]
+then
+    echo "running without babel-node"
+    node \
```

Comment:
so i'm not really sure why we still need babel-node here anyway since it all went through babel already and we're running the built version BUT i kept it and added an environment variable to allow you to choose not to use it because babel-node breaks breakpoints in some IDEs (which i agree is pretty weird)